clang/lib/Frontend/InitPreprocessor.cpp

```diff
@@ -159,26 +159,6 @@ static void DefineFloatMacros(MacroBuilder &Builder, StringRef Prefix,
   Builder.defineMacro(DefPrefix + "MIN__", Twine(Min)+Ext);
 }
 
-
```

Comment:
They were moved to `include/clang/Frontend/Utils.h` in flang-compiler/flang-driver@c51f89679135f84675f492d560ec5535c2000cfe, presumably because the functions are needed in `lib/Driver/ToolChains/Flang.cpp`.